packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -190,27 +191,46 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
           </Form>
         </Panel>
         {!isLoadingPeerProfile && isValidURL && (
-          <Panel header={t('peerList.panels.information')}>
-            <Panel
-              bordered={true}
-              style={{
-                height: '200px',
-                marginBottom: '20px',
-                backgroundSize: 'cover',
-                backgroundImage: `url(${
-                  profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
-                }`
-              }}
-            />
-            <DescriptionList>
-              <DescriptionListItem label={t('peerList.panels.name')}>
-                {profile?.name}
-              </DescriptionListItem>
-              <DescriptionListItem label={t('peerList.panels.themeColor')}>
-                {profile?.themeColor}
-              </DescriptionListItem>
-            </DescriptionList>
-          </Panel>
+          <>
+            <Panel header={t('peerList.panels.information')}>
+              <Panel
+                bordered={true}
+                style={{
+                  height: '200px',
+                  marginBottom: '20px',
+                  backgroundSize: 'cover',
+                  backgroundImage: `url(${
+                    profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
```

Comment:
rather a general remark: the placeholder is an interesting solution but I can imagine that there is also a certain downside to this: if the placeholder-provider changes something on their side we will have an error here. Also I'm not sure if we want to give free publicity to `HTML.com` ;)  

**@others:** what do you guys think?

Reply:
I do agree with you Pascal, I guess we may have a placeholder in assets(even though it will continue to make the project heavier). But I haven't worked on this part, it's Eslint formatting :smiley:

Reply:
@bishoymelek is this <> fragment is needed? Else this whole block wouldn't show up in the diff.